recipes/qt/bld.bat

```diff
@@ -0,0 +1,28 @@
+
+echo y | configure -prefix %LIBRARY_LIB%\qt4 ^
+                   -libdir %LIBRARY_LIB% ^
+                   -bindir %LIBRARY_LIB%\qt4\bin ^
+                   -headerdir %LIBRARY_INC%\qt4 ^
+                   -datadir %LIBRARY_LIB%\qt4 ^
+                   -L %LIBRARY_LIB% ^
+                   -I %LIBRARY_INC% ^
+                   -release ^
+                   -fast ^
+                   -no-qt3support ^
+                   -nomake examples ^
+                   -nomake demos ^
+                   -nomake docs ^
+                   -opensource ^
+                   -openssl ^
+                   -webkit ^
+                   -system-libpng ^
+                   -system-zlib ^
+                   -system-libtiff ^
+                   -system-libjpeg
+if errorlevel 1 exit 1
+                
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
```

Comment:
My `bld.bat` wasn't working properly anyway so you will save me some time trying to work out what is going on :+1: